### DISCUSSION TITLE
Will a non-clustered scheduler cause corruption if it's only managing the schedule and not executing jobs

### DISCUSSION THREAD

**Discussion by CrossBound** [quartznet/quartznet#1038] [Q&A] [answered]
Documentation under the ado.net section says to never **_start_** 2 non-clustered schedulers together that are the same in name, but the best practices page just says not to point 2 schedulers to the same database if they have the same name. 

In my case I want a Windows service to be running the jobs in the background but I want a separate web application which allows administration of the schedule. Will this cause corruption if my web application is not actually executing the jobs? Or will they still have to be setup as a cluster?

**Comment by lahma** [comment, marked as answer]
Clustered mode ensures that there's proper synchronization (locking) in place when retrieving triggers/jobs and updating their status in database. So when you alter the database you need to have the clustered mode on, otherwise you could basically delete a trigger from web while it's running on the other node - it would error the running node when it tries to return the trigger.

There's a slight performance impact as locks are then handled on DB side, but that shouldn't be that bad unless you have some ultra-high load scenario - which would warrant for more nodes anyway.

Just make sure you configure your web with ZeroSizeThreadPool and don't start it, then it's ready for maintenance mode.
